src/ts/show.ts

```diff
@@ -25,17 +25,29 @@ $(document).ready(function(){
   }
 
   let entityType = EntityType.Experiment;
-  if ($('#type').data('type') === 'items') {
+  if (about.type === 'items') {
     entityType = EntityType.Item;
   }
 
   const EntityC = new EntityClass(entityType);
 
-  // CREATE EXPERIMENT with shortcut
-  key($('#shortcuts').data('create'), function() {
-    EntityC.create('0').then(json => {
-      window.location.href = `experiments.php?mode=edit&id=${json.res}`;
-    });
+  // CREATE EXPERIMENT or DATABASE item with shortcut
```

Comment:
More or less copied from common.ts. Ideally this could be put in a function and reused but that will need a more rigorous overhaul of the surrounding JS.